00_find_bad_subs.py

```diff
@@ -0,0 +1,70 @@
+"""finds bad subjects and saves them as json.
+
+based on either sensitvity index d'
+or binomial test against chance.
+the latter works bc of a clever task design
+To reproduce, you have to delete the bad subs from config.
```

Comment:
>  the latter works bc of a clever task design

This comment doesn't really help (not me in any case :rofl:) --> what's clever about the task design that makes this work? That'd be a helpful comment, I think --- if you think writing it out would be overkill, you could also delete the current sentence.

Also: Let's try to write approximately as we would in a paper:

- bc --> because
- using capitalization etc.

Because we want people other than ourselves to read this code